Extending/Packages/UmbPack/index.md

```diff
@@ -0,0 +1,174 @@
+---
+versionFrom: 7.0.0
+meta.Title: "UmbPack"
+meta.Description: "How to use the UmbPack tool to deploy package versions to Our"
+---
+
+# UmbPack
+
+UmbPack is an open source NuGet tool that can deploy packages to Our, and also help you pack your files into a package. 
+
+:::note
+You still need to create the package on Our before the tool can be used. Once it has been set up you will then be able to add new versions of the package zip files using UmbPack
+:::
+
+## Creating an API key on Our.umbraco.com
+
+To create an API key to use with the UmbPack tool you should first log in with your user and go to the [manage packages section](https://our.umbraco.com/member/profile/packages/).
+There you will find a button under each package to manage API keys:
+
+![Api keys button](images/apiKeysButton.png)
+
+To generate a key, you need to add a description, when you click add it will show you your key. Be aware that as soon as you refresh the page the key will be gone, and if you lose it you will have to generate a new one.
```

Comment:
Wasn't really sure what this sentence meant - I've tried to rewrite it slightly. Does it still hold the intended meaning?

```suggestion
To generate a key, you need to add a description. When you click "Add" it will show you your key. Be aware that as soon as you refresh the page the key will be gone, and if you lose it you will have to generate a new one.
```